package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "move-dance-feel",
+  "version": "1.0.0",
+  "description": "This is the 3rd Project for Founder &amp; Coder's Founders Program",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/index.js",
+    "lint": "./node_modules/.bin/eslint ./src/**/*.js",
+    "lint-fix": "npm run lint --fix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fac-graduate-programme/Move-Dance-Feel.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
just a tiny change, our licence is actually MIT (it was the one mentioned in a contract me and Dylan had to sign), 
but looking good! <3 linting